### DISCUSSION TITLE
checkbox with pagination

### DISCUSSION THREAD

**Discussion by asinghjs** [livewire/livewire#8630] [1. Help]
hey there,
i am using laravel livewire 3 with pagination and checkbox . when i check the checkbox it set the value in array that is ok. but when i switch page to next it show checked checkbox which i have checked in previous page. Although server side value are remain same which i have checked. it should not be check in new page bcz that checkbox ids not exist in array.
pls help me with this.

**Comment by joshhanley** [comment]
@Iweb-tech Please share your code 🙂

**Comment by asinghjs** [comment]
`<div class="form-check">
                                        <input
                                            wire:model.live="selectedRows"
                                            type="checkbox"
                                            class="form-check-input @if($row->status == 0) transfer-checks border-secondary @endif"
                                            id="usersDataCheck{{$i}}" name="payout_check[]" value="{{$row->id}}"
                                            @if($row->status != 0) disabled @endif
                                        >
                                        <label class="form-check-label" for="usersDataCheck{{$i}}"></label>
                                    </div>`
                                    it bind in server component with
                                     public array $selectedRows = [];
<img width="1430" alt="Screenshot 2024-07-05 at 5 50 36 PM" src="https://github.com/livewire/livewire/assets/13196825/09135b69-0519-4feb-a449-04dcfc879d8b">
see in screenshot i have checked 3 checkbox in page one.also showing value of selected checkbox behind the reject button. but current page is second and checkbox are checked. which should not auto checked. this is the main issue.

**Comment by asinghjs** [comment]
@joshhanley sorry for my mistake . i did not add key to row of table . thats why it did not identify the unique row. i learn this from your video tutorial again. but thanks for your kind response. 🙏

**Reply by joshhanley** [reply]
@Iweb-tech ah no worries! In future when you share code, make sure to share more than just the issue as context matters (like adding in the foreach for example) as someone would have seen the missing key 🙂
